src/modules/esl-media/core/esl-media.ts

```diff
@@ -99,6 +100,8 @@ export class ESLMedia extends ESLBaseElement {
   @boolAttr() public playsinline: boolean;
   /** Allows play resource only in viewport area */
   @boolAttr() public playInViewport: boolean;
+  /** Viewport intersection marker */
+  @boolAttr() public inViewport: boolean;
```

Comment:
```suggestion
  @boolAttr({readonly: true}) public inViewport: boolean;
```
+move to readonly accessors declararion